tensorboard/plugins/hparams/tf_hparams_sessions_pane/tf-hparams-sessions-pane.html

```diff
@@ -70,7 +70,7 @@
               </a>
             </template>
             <template is="dom-if" if="[[helpUrl]]">
-              <a href$="[[helpUrl]]">
+              <a href$="[[helpUrl]]" target="_blank">
```

Comment:
add `rel="noopener noreferrer"`, please, to prevent `window.opener` hijacking

Reply:
I foresaw this happening :) I searched for target="_blank" and saw places we didn't do that so I, inside, thought 'hey! we don't care about rel noreferrer!'

Regardless, I will fix it.

Reply:
Thanks; opened <https://github.com/tensorflow/tensorboard/pull/2516>.